.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "eslint.options": {
+    "extensions": [".ts", ".html"]
+  },
+  "eslint.validate": [
+    "javascript",
+    "javascriptreact",
+    "typescript",
+    "typescriptreact",
+    "html"
+  ]
+}
```

Comment:
I imagine this is coming from some template, but it is curious that it validates javascript but it doesn't expect the ".js" extension.

Reply:
Yes, there's a link in the *Other information* section.